src/main/java/org/hawkular/client/android/activity/AlertsActivity.java

```diff
@@ -75,11 +77,50 @@ private void showProgress() {
     }
 
     private void setUpAlerts(List<Alert> alerts) {
-        list.setAdapter(new AlertsAdapter(this, alerts));
+        list.setAdapter(new AlertsAdapter(this, this, alerts));
 
         hideProgress();
     }
 
+    @Override
+    public void onAlertMenuClick(View alertView, int alertPosition) {
+        showAlertMenu(alertView, alertPosition);
+    }
+
+    private void showAlertMenu(final View alertView, final  int alertPosition) {
+        PopupMenu alertMenu = new PopupMenu(this, alertView);
+
+        alertMenu.getMenuInflater().inflate(R.menu.menu_alerts, alertMenu.getMenu());
+
+        alertMenu.setOnMenuItemClickListener(new PopupMenu.OnMenuItemClickListener() {
+            @Override
+            public boolean onMenuItemClick(MenuItem menuItem) {
+                switch (menuItem.getItemId()) {
+                    case R.id.menu_resolve:
+                        removeAlert(alertPosition);
+                        return true;
+
+                    case R.id.menu_acknowledge:
+                        removeAlert(alertPosition);
```

Comment:
The rebase process messed things up a little. As I mentioned at the description there is no actual action processing at this point due to [API conflicts](http://lists.jboss.org/pipermail/hawkular-dev/2015-June/001191.html).